templates/books-index.html

```diff
@@ -0,0 +1,30 @@
+<!-- header -->
+<div class="jumbotron">
+  <div class="container">
+    <div class="col-md-6">
+      <h1 class="yay">YAY BOOKS!</h1>
+    </div>
+    <div class="col-md-4 col-md-offset-2">
+      <img class="books-image" src="../public/images/books.png">
+    </div>
+  </div>
+</div>
+
+<!-- search bar -->
+<div class="container-fluid">
+  <div class="col-md-12 search-bar">
+    <div class="col-md-3" display="block">
+      <input type="text" class="form-control" ng-model="searchText" placeholder="Search Books">
```

Comment:
It's better to use labels rather than placeholder text to explain the use of an input; recall that placeholder text isn't always shown to the user. https://www.nngroup.com/articles/form-design-placeholders/